app/js/controllers/main.js

```diff
@@ -92,4 +117,24 @@ export default class MainController extends BaseController {
         console.error('Error while subscribing to notifications:', err);
       });
   }
+
+  /**
+   * Will init the `voiced` property, which controls whether a notification is
+   * spoken.
+   *
+   * This works by using a specific hash while loading or running the
+   * application.
+   * Recognized syntaxes are: voiced=1/true/0/false
+   */
+  [p.initVoiced]() {
+    const forcedMatch = location.hash.match(/\bvoiced=(.+?)(&|$)/);
+
+    if (forcedMatch) {
+      const matchedValue = forcedMatch[1];
+      const willSpeak = matchedValue === '1' || matchedValue === 'true';
```

Comment:
Can you call it `voiced` as it's better for consistency to call the variable the same as the property and the hashtag and it avoids possible confusion?